### ISSUE TITLE
How to ignore `nan` when using `cross_value`?

### ISSUE THREAD

**Issue by GF-Huang** [twopirllc/pandas-ta#500]
Or specify a default value as result when compare a `nan`.

**Comment by GF-Huang** [comment]
Here is my use case. The first 6 rows of `CrossDown80` expected value should be `false`, but `true` here.

![image](https://user-images.githubusercontent.com/4510984/157830132-4c77d106-44fc-4ce5-ad85-2258267c5574.png)

![image](https://user-images.githubusercontent.com/4510984/157830279-da0a732f-e853-4d5d-802d-ba9202f78eba.png)

**Comment by twopirllc** [comment]
@GF-Huang,

Right.. RSI Crossings. I recommend checking out: https://github.com/twopirllc/pandas-ta/issues/406. Perhaps that might address the issue? 🤷🏼‍♂️  Lemme know.

KJ

**Comment by GF-Huang** [comment]
The `ta.xsignal` seems not meet my use case, which if I only need the entries, without exits.

![image](https://user-images.githubusercontent.com/4510984/157950400-5c605a64-9b47-4ce0-9ab5-9186b404e303.png)

**Comment by twopirllc** [comment]
@GF-Huang,

Sorry for the delay.

Unfortunate that _xsignals_, it does need some work and assistance on it would be great. 😎 

Regarding _cross_value_, perhaps @lluissalord may have suggestions as well. 🤷🏼‍♂️ 

KJ

**Comment by twopirllc** [comment]
Please see #901
